.github/workflows/renovate.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: renovatebot/github-action@v34.152.3
+      - uses: renovatebot/github-action@v34.152.5
```

Comment:
Why not pin to @v34 to automatically allow minor and patch updates? Most of our actions are pinned to the major version only.